libs/build_mapl.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -eux
+
+name="mapl"
+repo=${1:-${STACK_mapl_repo:-"GEOS-ESM"}}
+version=${2:-${STACK_mapl_version:-"main"}}
+
+# Hyphenated version used for install prefix
+compiler=$(echo $HPC_COMPILER | sed 's/\//-/g')
+mpi=$(echo $HPC_MPI | sed 's/\//-/g')
+id=${version//\//-}
+version_install=$repo-$id
+
+if $MODULES; then
+  set +x
+  source $MODULESHOME/init/bash
+  module load hpc-$HPC_COMPILER
+  module load hpc-$HPC_MPI
+  module load hpc-$HPC_PYTHON
```

Comment:
Is MAPL needing python in the build here?

Reply:
Yes

Reply:
where is MAPL using Python during build and installation?

This is a snippet from the `MAPL/CMakeLists.txt`
```
install (
   DIRECTORY Python
   DESTINATION lib
   USE_SOURCE_PERMISSIONS
   )
```
It is simply copying the contents of the `Python` directory to the installation location.

`hpc-$HPC_PYTHON` module is not required for building and installing MAPL.
Please remove.

Reply:
Done in 152b70e64a501f09f4dd9eab49d7fe92d4c2e2fa.